pkgs/development/tools/marksman/default.nix

```diff
@@ -0,0 +1,50 @@
+{ lib
+, fetchFromGitHub
+, buildDotnetModule
+, dotnetCorePackages
+}:
+
+buildDotnetModule rec {
```

Comment:
Nit: If possible, it would really be better to use the new [recursive attributes feature of `mkDerivation`](https://nixos.org/manual/nixpkgs/stable/#idm140737319947216), but I don't know if `buildDotnetModule` supports that yet.

Reply:
> but I don't know if `buildDotnetModule` supports that yet.

I don't think it does.

Reply:
Why would we want to use that here? Just to substitute version in the fetcher? The fetcher is FOD anyway, so it would only work when taking extra care to overwrite the fetcher which is not documented anywhere. More the opposite is that src is always overwritten in all examples. Also people tend to use e.g. pname all over the place which would catastrophically fail with finalAttrs.